.env.sample

```
@@ -0,0 +1,9 @@
+DATABASE_URL=“mysql://usersname:password@127.0.0.1:3306/database_name”
```

```suggestion
DATABASE_URL=“rdbms://usersname:password@iphost:port/database_name”
```
작성 잘해주셨습니다! 조금 더 정확히 작성하자면 .env.sample에서는 어떠한 것도 직접적으로 들어나는것이 아닌 중의적으로 작성되도록 해야합니다. 사용하는 rdbms가 mysql이고 localhost이고 , 3306(mysql port number) 라는 것을 나타내지 않고 첨부드린 것과 같이 수정될 수 있을 것 같습니다!